src/libcrun/linux.c

```diff
@@ -5713,3 +5719,26 @@ libcrun_update_intel_rdt (const char *ctr_name, libcrun_container_t *container,
 
   return resctl_update (name, l3_cache_schema, mem_bw_schema, err);
 }
+
+int
```

Comment:
nit: Could we add a comment describing why what we do works to detect a working directory outside the rootfs?

Reply:
sure, that is a good idea.